chapter_introduction/index_vn.md

```diff
@@ -1540,7 +1540,14 @@ Below is a very simple example of annotating a sentence
 with tags indicating which words refer to named entities.
 -->
 
-*dịch đoạn phía trên*
+**Gán thẻ và Phân tích cú pháp**.
+Đây là bài toán gán các thuộc tính cho chuỗi các từ.
+Nói cách khác, số lượng đầu vào và đầu ra là như nhau.
+Ví dụ, ta có thể muốn biết vị trí của động từ và chủ ngữ.
+Hoặc ta cũng có thể muốn biết từ nào là thực thể được đặt tên. <!-- hmm, thảo luận -->
```

Comment:
Mình thấy ví dụ của `named-entity` thường là "James", "University of Washington", "2006", etc, không biết dịch là `danh từ riêng` được không nhỉ? 
Edit: ah theo như ví dụ ở dưới thì các named-entity đó đều là danh từ riêng cả :))

Reply:
Anh @sonvx từ named-entity bên NLP dịch là gì nhỉ?

Reply:
tớ thấy định nghĩa của nó đúng là danh từ riêng đó (check cả trên wiki).

Reply:
okay.